### ISSUE TITLE
Where are firmwares for X8 2020 available?

### ISSUE THREAD

**Issue by Tomiz-opz** [wladimir-computin/FimiX8-SDFG#15]
json file didnt have links for 2020 version of X8. Good to have also 2020 version included.

**Comment by Witalka-Shelby** [comment]
I was also searching for the 2020 version and found that the json file has the 2020 version but this app is looking only for ''sysNameI18N'' and this why I think this app is mixing (I think so) some firmware files.. because when I search only for ''X8S2'' there are only 5 firmware files.. the zh_CN writing the name ''X8S2-飞控中继'' not like the en_US name "FC Relay"

also the fimi.com site has download links
https://www.fimi.com/media/Productattachments//u/p/upgrade_x8s2_cam_v2023b_1.zip

as we can see X8S2 in file name... but the problem now.. I do not have the skill to change this ...

**Comment by Tomiz-opz** [comment]
have you tried these ones ? https://cdn.awsusor0.fds.api.mi-img.com/bucket-firmware?/?list-type=2&prefix=&delimiter=

**Comment by Witalka-Shelby** [comment]
no this is a new link to me! but I am still do not know what to do :)..
for the FC I found a way to hex edit the firmware to flash any firmware also downgrade.. for the camera fimi support sent me files...

**Comment by Tomiz-opz** [comment]
> no this is a new link to me! but I am still do not know what to do :)..
> for the FC I found a way to hex edit the firmware to flash any firmware also downgrade.. for the camera fimi support sent me files...

Yes, Navi app for  Fimi X8 2020 queries firmwares from that site. Older firmwares can be found also there, you have just to know exact filename
